content/docs/studio/selfhosted/installation/ami.md

```diff
@@ -0,0 +1,143 @@
+# Installing Studio Self-hosted via the AMI
+
+1. Open the AWS Console
+
+2. Navigate to EC2 -> Instances
+
+3. Click **Launch instances**
+
+4. Provide a name for your EC2 instance
+
+![](/img/studio-selfhosted-ami-1.png)
+
+5. Select **studio-selfhosted** from the AMI catalog (owner: 260760892802)
+
+![](/img/studio-selfhosted-ami-2.png)
+
+6. Select an appropriate instance type. The minimum requirements are 16 GB RAM
+   and four vCPUs, and the recommended requirements are 32 GB RAM and eight
+   vCPUs. In this example, we’ll choose the `m6i.2xlarge` type:
+
+![](/img/studio-selfhosted-ami-3.png)
+
+7. Select an existing key pair to use or create an existing one. We recommend
+   ED25519 keys.
+
+![](/img/studio-selfhosted-ami-4.png) ![](/img/studio-selfhosted-ami-5.png)
+
+8. In network settings, use the default VPC or change it to a desired one. Under
+   the Firewall setting, create a new security group with SSH, HTTP, and HTTPS
+   access or use an existing one with the same level of access.
+
+![](/img/studio-selfhosted-ami-6.png)
+
+<admon type="warn">
+
+Please ensure that your VPC has connectivity to your SCM provider (Github,
+Gitlab, or Bitbucket)
+
+</admon>
+
+9. Configure the storage. We recommend a minimum of 50 GB block storage.
+
+![](/img/studio-selfhosted-ami-7.png)
+
+10. Launch the EC2 instance
+
+![](/img/studio-selfhosted-ami-8.png)
+
+11. Wait for the EC2 instance to become available. Now, copy its IP address.
+
+12. Open a terminal, and run the following commands:
+
+```shell
+$ export DOCKER_USERNAME=<Docker registry username>
+$ export DOCKER_PASSWORD=<Docker registry password>
+$ export EC2_INSTANCE=<EC2 instance hostname>
+
+$ ssh -i studio.pem ubuntu@$EC2_INSTANCE "kubectl create secret docker-registry iterativeai --namespace studio --docker-server=docker.iterative.ai --docker-username=$DOCKER_USERNAME --docker-password=$DOCKER_PASSWORD"
+```
```

Comment:
Let's ask users to store deploy Studio from Values on the Instance.

Reasons:
1. The `values.yaml` will not stand on the user(operator) host - this is a common issue in companies when the operator leaves a company, and all configs are lost with his computer🙃 
2. The command will be cleaner - it will just ask for the installation, will be easier to copy

My idea is  that the manual should have this workflow
```mermaid
flowchart TD;
     A[Instance is running] --> B[Create values.yaml]
    B --> C[Upload values.yaml to Instance]
    C  --> D[ssh to instance]
    D --> E[Create Secrets & Deploy studio]
    E --> F[Confirm it is working]
```

Also we will unify the docs with other pages e.g. TLS to merge values 😕 https://dvc-org-studio-selfhost-l2oxcs.herokuapp.com/doc/studio/self-hosting/configuration/tls

Reply:
@mjasion @jesper7  - it will be simpler to guide users to creating the `values.yaml` file on the instance (AMI).
In any case, the values can be "rescued" from an existing helm release always, see [`helm get values`](https://helm.sh/docs/helm/helm_get_values/)